CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
```

Comment:
How about we keep this to issue board and the discord for now. Issue board definitely is the right place to discuss the changes and bugs, but for more help and guidance for contribution we could use discord as well.

It gets easier for the maintainer to keep track of things

Reply:
Yeah sure, I'll add a discord link there so that users can directly join the discord group and ask whatever doubts they have in the channel itself ..

That's works right?

Reply:
Yea, issue and discord works. We can remove "email, or any other method" part

Reply:
Ok, cool 👍